tests/drivers/w5500/Makefile

```
@@ -0,0 +1,26 @@
+include ../Makefile.drivers_common
+
+USEMODULE += netdev_default
+USEMODULE += auto_init_gnrc_netif
+# Activate ICMPv6 error messages
+USEMODULE += gnrc_icmpv6_error
+# Specify the mandatory networking module for a IPv6 routing node
+USEMODULE += gnrc_ipv6_router_default
```

```suggestion
USEMODULE += gnrc_ipv6_default
```

no need to be a router